packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -508,9 +511,6 @@ def thread_request():
     assert set(expected_uuids) == set(received_uuids)
 
 
-@pytest.mark.skipif(not lb_enabled(), reason='Load Balancer disabled')
-@pytest.mark.xfail(test_helpers.expanded_config.get('security') == 'strict',
```

Comment:
This is a change from `xfail` to `skip` because it is complex to achieve `xfail` within a function, and I see little value.